.gitpod/readme.md

```diff
@@ -0,0 +1,49 @@
+# Welcome to Starport ✨
+
+Starport is an easy to use CLI tool for creating sovereign blockchains. Blockchains created with Starport use Cosmos SDK, the world's most widely-used blockchain application framework.
+
+In front of you is a browser-based development environment. At the bottom of the screen you can see a terminal window, it has `starport` binary pre-installed.
```

Comment:
```suggestion
In this browser-based development environment, the terminal window is in the lower part of the window. The `starport` binary is pre-installed and ready to use on the command line.
```

Reply:
tricky to describe positional UI, per https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/l/lower-left-lower-right

Reply:
That's really good!